Update list-assets.json

kaimeinke opened the pull request:
Included the pools of a bad actor. This refers to https://github.com/oceanprotocol/list-purgatory/issues/27 by AgentK77 and https://github.com/oceanprotocol/list-purgatory/pull/25 by me.

kaimeinke committed:
Update list-assets.json

Included the pools of a bad actor. This refers to https://github.com/oceanprotocol/list-purgatory/issues/27 by AgentK77 and https://github.com/oceanprotocol/list-purgatory/pull/25 by me.

kremalicious committed:
Merge branch 'main' into patch-2

daneklustig reviewed (changes requested):
Please use one of the categories for putting datasets into purgatory: rug pull / sensitive data / impersonation / ip infringement / violation

kaimeinke committed:
Update list-assets.json

Changed the reasons according to the list of categories. Categories for putting datasets into purgatory: rug pull / sensitive data / impersonation / ip infringement / violation

kaimeinke commented:
Hi @daneklustig , I updated the categories accordingly.
